index.html

```diff
@@ -174,26 +226,17 @@ <h3 class="name-project">project name goes here</h3>
                 ><img src="./assests/images/Union.png" alt="arrow image"
               /></span>
             </button>
+            <button class="button-project768">
+              See this project
+              <span
+                ><img src="./assests/images/Union2.png" alt="arrow image"
+              /></span>
+            </button>
           </article>
-        </section>
-
-        <section class="about-container">
-          <h2 class="about-header">About</h2>
-          <p class="about-paragraph">
-            Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
-            eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim
-            ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut
-            aliquip ea eiusmod
-          </p>
-          <button class="cv-button">
-            Get my cv
-            <span id="cv-span">
-              <img
-                src="./assests/images/ic_download.svg"
-                alt="get my cv icon"
-              />
-            </span>
-          </button>
+        </article>
+      </section>
+      <section class="about-package">
+        <article class="package">
           <section class="languages">
             <h3 class="lang-header">languages</h3>
```

Comment:
- [ ] Please place some left, right margins for this section to match the design.

### Your design

![image](https://user-images.githubusercontent.com/83803414/160181156-9432deb0-3f52-4fc5-bba9-e39007512acd.png)

### Figma design

![image](https://user-images.githubusercontent.com/83803414/160181179-54f8ca8d-2233-44bd-8a1d-c086e0d9711a.png)